source/documentation/modules/math.html.md.erb

```diff
@@ -0,0 +1,494 @@
+---
+title: sass:math
+---
+
+<%= partial '../snippets/built-in-module-status' %>
+
+## Variables
+
+<% function 'math.$e' do %>
+  Equal to the value of the mathematical constant `e`.
+
+  <% example(autogen_css: false) do %>
+    @debug math.$e; // 2.7182818285
+    ===
+    @debug math.$e // 2.7182818285
+  <% end %>
+<% end %>
+
+
+<% function 'math.$pi' do %>
+  Equal to the value of the mathematical constant `pi`.
+
+  <% example(autogen_css: false) do %>
+    @debug math.$pi; // 3.1415926536
+    ===
+    @debug math.$pi // 3.1415926536
+  <% end %>
+<% end %>
+
+
+## Bounding Functions
+
+<% function 'math.ceil($number)', 'ceil($number)', returns: 'number' do %>
+  Rounds `$number` up to the next highest whole number.
+
+  <% example(autogen_css: false) do %>
+    @debug math.ceil(4); // 4
+    @debug math.ceil(4.2); // 5
+    @debug math.ceil(4.9); // 5
+    ===
+    @debug math.ceil(4)  // 4
+    @debug math.ceil(4.2)  // 5
+    @debug math.ceil(4.9)  // 5
+  <% end %>
+<% end %>
+
+
+<% function 'math.clamp($min, $number, $max)', returns: 'number' do %>
+  Clamps `$number` in between `$min` and `$max`.
```

Comment:
Be more explicit about this. If it were clear to a reader what "clamps" meant, they probably wouldn't need to read the documentation :smile:.

Reply:
So you're telling me

> clamp /klamp/ *verb*:
> to clamp

isn't good enough? 😛 

Fixed.